src/server/pfs/server/obj_block_api_server.go

```diff
@@ -62,55 +72,103 @@ func newObjBlockAPIServer(dir string, cacheBytes int64, objClient obj.Client) (*
 		return nil, err
 	}
 	oneCacheShare := cacheBytes / (objectCacheShares + tagCacheShares + objectInfoCacheShares)
-	server := &objBlockAPIServer{
+	s := &objBlockAPIServer{
 		Logger:           protorpclog.NewLogger("pfs.BlockAPI.Obj"),
 		dir:              dir,
 		localServer:      localServer,
 		objClient:        objClient,
 		objectIndexes:    make(map[string]*pfsclient.ObjectIndex),
 		objectCacheBytes: oneCacheShare * objectCacheShares,
 	}
-	server.blockCache = groupcache.NewGroup("block", cacheBytes,
-		groupcache.GetterFunc(server.blockGetter))
-	server.objectCache = groupcache.NewGroup("object", oneCacheShare*objectCacheShares,
-		groupcache.GetterFunc(server.objectGetter))
-	server.tagCache = groupcache.NewGroup("tag", oneCacheShare*tagCacheShares,
-		groupcache.GetterFunc(server.tagGetter))
-	server.objectInfoCache = groupcache.NewGroup("objectInfo", oneCacheShare*objectInfoCacheShares,
-		groupcache.GetterFunc(server.objectInfoGetter))
-	return server, nil
+	s.objectCache = groupcache.NewGroup("object", oneCacheShare*objectCacheShares, groupcache.GetterFunc(s.objectGetter))
+	s.tagCache = groupcache.NewGroup("tag", oneCacheShare*tagCacheShares, groupcache.GetterFunc(s.tagGetter))
+	s.objectInfoCache = groupcache.NewGroup("objectInfo", oneCacheShare*objectInfoCacheShares, groupcache.GetterFunc(s.objectInfoGetter))
+	go s.watchGC(etcdAddress)
```

Comment:
So right now there's no way to cancel this goro? That's probably ok given that this is normally supposed to run for the life of the problem.